### ISSUE TITLE
Error: unhandled exception: invalid integer: 130.000.000

### ISSUE THREAD

**Issue by hugosenari** [xyb/googlesearch.nim#3]
When running google distance it fails with parseInt

```bash
googledistance 'praca da se, sao paulo, sp' 'rua domingos de morais, sao paulo, sp'
```
> The Normalized Google Distance (NGD) of praca da se, sao paulo, sp and rua domingos de morais, 236, sao paulo, sp is:
strutils.nim:1072        parseInt
Error: unhandled exception: invalid integer: 130.000.000 [ValueError]

**Comment by xyb** [comment]
@hugosenari thank you for the report, I have released a bug-fix version v0.4.2, please give it a try.

**Comment by hugosenari** [comment]
To be honest, I was expecting another kind of 'distance' :rofl: 
(that's why my queries are locations)

Now the same query as before returns `inf`

And `'praca da se' 'rua domingos de morais'` returns `0.000`

I'm not sure if that inf is right, surely the original issue was resolved. :+1:
